membership-attribute-service/app/controllers/AccountController.scala

```diff
@@ -333,7 +333,7 @@ class AccountController(commonActions: CommonActions, override val controllerCom
       // subscription so we can reuse it and avoid a call to Zuora
       nonGiftSubs.find(_.subscription.name == subscriptionName) match {
         case Some(contactAndSubscription) => Future.successful(Some(contactAndSubscription.subscription))
-        case _ => subscriptionService.get[AnyPlan](subscriptionName, isActiveToday = true)
+        case _ => subscriptionService.get[AnyPlan](subscriptionName, isActiveToday = false)
```

Comment:
Would it be worth adding a comment to explain why this is false rather than true?

Reply:
Maybe, there is an extensive Javadoc comment on the method itself:
<img width="875" alt="Screen Shot 2022-01-12 at 14 49 00" src="https://user-images.githubusercontent.com/181371/149163059-65ca1aea-9763-4964-b055-487d3857d840.png">

Do you think I should add something else?

Reply:
Maybe, there is an extensive Javadoc comment on the method itself:
<img width="875" alt="Screen Shot 2022-01-12 at 14 49 00" src="https://user-images.githubusercontent.com/181371/149163059-65ca1aea-9763-4964-b055-487d3857d840.png">

Do you think I should add something else?

Reply:
I added something in.

Reply:
Thanks.  That helps to explain why it works for gifts, which I don't think is clear in the javadoc.  Although I hadn't read the javadoc anyway.